src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/SymUnmanagedReaderExtensions.cs

```diff
@@ -21,56 +21,59 @@ internal static class SymUnmanagedReaderExtensions
             var importStringGroups = reader.GetCSharpGroupedImportStrings(methodToken, methodVersion, out externAliasStrings);
             Debug.Assert(importStringGroups.IsDefault == externAliasStrings.IsDefault);
 
-            if (importStringGroups.IsDefault)
+            ArrayBuilder<ImmutableArray<ImportRecord>> importRecordGroupBuilder = null;
+            if (!importStringGroups.IsDefault)
             {
-                return default(MethodDebugInfo);
-            }
-
-            var importRecordGroupBuilder = ArrayBuilder<ImmutableArray<ImportRecord>>.GetInstance(importStringGroups.Length);
-            foreach (var importStringGroup in importStringGroups)
-            {
-                var groupBuilder = ArrayBuilder<ImportRecord>.GetInstance(importStringGroup.Length);
-                foreach (var importString in importStringGroup)
+                importRecordGroupBuilder = ArrayBuilder<ImmutableArray<ImportRecord>>.GetInstance(importStringGroups.Length);
+                foreach (var importStringGroup in importStringGroups)
                 {
-                    ImportRecord record;
-                    if (NativeImportRecord.TryCreateFromCSharpImportString(importString, out record))
+                    var groupBuilder = ArrayBuilder<ImportRecord>.GetInstance(importStringGroup.Length);
+                    foreach (var importString in importStringGroup)
                     {
-                        groupBuilder.Add(record);
-                    }
-                    else
-                    {
-                        Debug.WriteLine($"Failed to parse import string {importString}");
+                        ImportRecord record;
+                        if (NativeImportRecord.TryCreateFromCSharpImportString(importString, out record))
+                        {
+                            groupBuilder.Add(record);
+                        }
+                        else
+                        {
+                            Debug.WriteLine($"Failed to parse import string {importString}");
+                        }
                     }
+                    importRecordGroupBuilder.Add(groupBuilder.ToImmutableAndFree());
                 }
-                importRecordGroupBuilder.Add(groupBuilder.ToImmutableAndFree());
             }
 
-            var externAliasRecordBuilder = ArrayBuilder<ExternAliasRecord>.GetInstance(externAliasStrings.Length);
-            foreach (string externAliasString in externAliasStrings)
+            ArrayBuilder<ExternAliasRecord> externAliasRecordBuilder = null;
+            if (!externAliasStrings.IsDefault)
```

Comment:
I'm not sure what would happen if we had extern aliases without import strings.  I suspect that's what the original if was guarding against.  Perhaps this if should be inside the first one?  You'd have to check the consumers of ```MethodDebugInfo```.